README.md

```diff
@@ -15,7 +15,17 @@ except according to the terms contained in the LICENSE file.
 [![License](https://img.shields.io/badge/license-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Build status](https://circleci.com/gh/getodk/central-frontend.svg?style=shield)](https://circleci.com/gh/getodk/central-frontend)
 
-ODK Central Frontend uses Vue.js to provide the frontend for [ODK Central](https://github.com/getodk/central). It is currently [under development](https://forum.getodk.org/t/whats-coming-in-central-over-the-next-few-years/19677).
+ODK Central is the [ODK](https://getodk.org/) server. It manages user accounts and permissions, stores form definitions, and allows data collection clients like ODK Collect to connect to it for form download and submission upload. ODK Central Frontend provides the frontend for ODK Central using Vue.js.
+
+This repository contains the code for the frontend of ODK Central. The [`central-backend`](https://github.com/getodk/central-backend) repository contains the code for the backend API server. The [`central`](https://github.com/getodk/central) repository contains the Docker code for building and running a production Central stack. You can find release notes in the `central` repository.
+
+The `master` branch of this repository reflects ongoing development for the next version of ODK Central. It may or may not be in sync with the `master` branch of the `central-backend` repository. For the latest stable version, see the [release tags](https://github.com/getodk/central-frontend/releases).
```

Comment:
I'd almost want to bold this. We should also have corresponding language in `central-backend` -- it would be bad to deploy from `master` now, for example (no backups, expiring App Users, etc).